implement briefing/drop camera lighting controls to match moodTags for contract (#1)
BlueWinds/EnvironmentalDesignMasks: README.md

### Basic Fields
Only two fields are required:
- `ID`: A human readable name for the mood. Must be globally unique. This may be visible when selecting the mood for a skirmish, but is otherwise not used in the UI.

> - `baseMood`: The ID of the mood this one inherets from. Any optional properties ~~(other than `moodTags`)~~ not defined will be copied from the baseMood into this one. This can be either any mood from the base game (see the log file after loading into the game for a list), or another custom Mood already loaded.

Reviewer: ```suggestion
- `baseMood`: The ID of the mood this one inherets from. Any optional properties not defined will be copied from the baseMood into this one. This can be either any mood from the base game (see the log file after loading into the game for a list), or another custom Mood already loaded.
```

    }
  ]
  ```

> - `moodTags`: A list of biomes for which this Mood can be selected in the Skirmish bay. ~~These tags have no effect outside the skirmish bay.~~ These moodTags also control the lighting in the leopard/deployment screen as defined by `briefingCameraTimeTags`.

Reviewer: ```suggestion
- `moodTags`: A list of biomes for which this Mood can be selected in the Skirmish bay. These moodTags also control the lighting in the leopard/deployment screen as defined by `briefingCameraTimeTags`, and determine which biomes a mood will be visible for in the skirmish bay.
```

  - `Horizontal`
  - `Vertical`
  - `HorizAndVert`

## Leopard/Deployment screen

> The lighting/time of day shown during the deployment screen is randomly chosen by default. EDM now allows you to control the lighting profile and match it to the appropriate moodTags for the current contract. EDM ships with appropriate default settings, but mod authors may wish to tweak them. Importantly, **the first matching tag found will control the lighting used.** If multiple lighting profiles are available for a given tag (the list of integers in the below setting), one will be chosen at random.

Reviewer: ```suggestion
The lighting/time of day shown during the deployment screen is randomly chosen by default. EDM now allows you to control the lighting profile and match it to the appropriate moodTags for the current contract. EDM ships with appropriate default settings, but mod authors may wish to tweak them.

EDM looks at the chosen mood's `moodTags`, comparing them in order to `briefingCameraTimeTags`. The **first** matching tag is used, and a lighting profile randomly selected from that list.
```

e.g for the following in settings.json:

```
"briefingCameraTimeTags": {
		"mood_timeMorning": [0,	1],
		"mood_timeSunset": [0, 1, 3	],
		"mood_timeAfternoon": [2, 4],
		"mood_timeNoon": [2, 4],

> 		"mood_timeDay": [0,	1, 2, 4],

Reviewer: ```suggestion
		"mood_timeDay": [0, 1, 2, 4],
```
